js/upbit.js

```diff
@@ -802,6 +802,10 @@ module.exports = class upbit extends Exchange {
             request['unit'] = numMinutes;
             method += 'Unit';
         }
+        if (since !== undefined) {
+            // convert `since` to `to` value
+            request['to'] = this.iso8601 (since + (timeframePeriod * limit * 1000));
```

Comment:
you have to use `this.sum` here since the `+` is ambiguous in php it can mean `.` or `+`

Reply:
@0xsaika thx! Please see the details here: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#adding-numbers-and-concatenating-strings

Reply:
Thx for the review! I'll fix it, hold on sec.

Reply:
> @0xsaika thx! Please see the details here: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#adding-numbers-and-concatenating-strings

Oops.. I missed that part ;) btw I commit the fix. thx for the notice!

Reply:
@0xsaika thx for the fix! )